addons/payment/__manifest__.py

```diff
@@ -1,31 +1,39 @@
-# -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 {
     'name': 'Payment Acquirer',
+    'version': '2.0',
     'category': 'Hidden',
     'summary': 'Base Module for Payment Acquirers',
-    'version': '1.0',
     'description': """Payment Acquirer Base Module""",
     'depends': ['account'],
     'data': [
         'data/account_data.xml',
         'data/payment_icon_data.xml',
         'data/payment_acquirer_data.xml',
         'data/payment_cron.xml',
-        'views/payment_views.xml',
-        'views/account_payment_views.xml',
-        'views/account_invoice_views.xml',
-        'views/payment_acquirer_onboarding_templates.xml',
-        'views/payment_templates.xml',
-        'views/payment_portal_templates.xml',
+
         'views/assets.xml',
+        'views/payment_portal_templates.xml',
+        'views/payment_templates.xml',
+
+        'views/account_invoice_views.xml',
+        'views/account_payment_views.xml',
+        'views/payment_acquirer_views.xml',
+        'views/payment_icon_views.xml',
+        'views/payment_transaction_views.xml',
+        'views/payment_token_views.xml',  # Depends on `action_payment_transaction_linked_to_token`
         'views/res_partner_views.xml',
+
+        # TODO TBE From what I saw of the functional flow, a user should never have to edit a tx
+        #          manually. We should maybe remove the perm_write and perm_create rights for users
+        #          now that txs are created with sudo?
```

Comment:
Yup, that a good practice. If a model doesn't need to be edited/created by a user manually. Then all the operations in the controllers should be done using `sudo` (while making sure its secure).

So, if you are 100% sure that no flow is broken after removing the `create` `write` rights, then I'm ok :)

Reply:
I'm not 100% sure but then, will we ever be sure?

I'd go for removing the rights for the reasons listed above and below but I'd understand if you have a diverging opinion about this.